src/containers/AdminDashboard/index.js

```diff
@@ -177,9 +195,23 @@ export class AdminDashboard extends Component {
     return true;
   }
 
-  validateDelete() {
+  validateDeleteForum() {
     const { forumId } = this.state;
-    if (forumId !== '') return false;
+    let notEmpty = false;
```

Comment:
@Unesdala this code is no better than how it was originally, just adds more variables, please put it back how it was before